src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -28,11 +44,17 @@ export class PropertyTemplateOutline extends Component {
     return classNames
   }
 
-  handleAddClick = (event) => {
+  handleAddClick = (resourceTemplate) => (event) => {
```

Comment:
I'm unfamiliar with the double arrow syntax here. Can you add a comment explaining what's going on here?

Reply:
This is an example of functional currying in Javascript. Made a comment in the code.